mono/metadata/threadpool-ms.c

```diff
@@ -501,10 +537,12 @@ worker_park (void)
 	mono_mutex_lock (&threadpool->active_threads_lock);
 
 	if (!mono_runtime_is_shutting_down ()) {
+		MonoInternalThread *thread = mono_thread_internal_current ();
```

Comment:
It is, on line 543 and 547.

On Wed, 30 Sep 2015 20:56 Rodrigo Kumpera <notifications@github.com> wrote:

> In mono/metadata/threadpool-ms.c
> <https://github.com/mono/mono/pull/2100#discussion_r40842976>:
>
> > @@ -501,10 +537,12 @@ worker_park (void)
> >  	mono_mutex_lock (&threadpool->active_threads_lock);
> >
> >  	if (!mono_runtime_is_shutting_down ()) {
> > +		MonoInternalThread *thread = mono_thread_internal_current ();
>
> This variable is not used anywhere
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mono/mono/pull/2100/files#r40842976>.
>